src/styles/card.scss

```diff
@@ -0,0 +1,140 @@
+@use './variables' as *;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+  margin: 0 auto;
+
+  @media (min-width: 0) {
+    & {
+      grid-template-columns: repeat(1, 200px);
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  grid-auto-flow: column;
+  transition: 0.3s ease-in;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    text-align: center;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-family: $card-main-font;
+    font-weight: 500;
+    color: $card-text-main-color;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The transform scale value should be written as a decimal (1.2) instead of a percentage (120%). Using 'scale(1.2)' is the correct way to specify scaling by 20%.